include/tvm/relay/error.h

```diff
@@ -20,8 +20,14 @@ struct InternalError : Error {
   explicit InternalError(const std::string &msg) : Error(msg) {}
 };
 
-// TODO(@jroesch): we should change spanned errors to report
-// errors against the Environment, inverting control to error definition.
+// FIX, we should change spanned errors to have a method which allow them to
+// report on the Environment, inverting control to error definition.
+struct SpannedError {
```

Comment:
this seems to be not relevant to this PR, consider to remove this for now

Reply:
removed